docs/csharp/tutorials/console-teleprompter.md

```diff
@@ -111,14 +112,14 @@ using System.IO;
 ```
 
 The `IEnumerable<T>` interface is defined in the 
-`System.Collections.Generic` namespace. The File class is defined in the
-`System.IO namespace`.
+`System.Collections.Generic` namespace. The `File` class is defined in the
```

Comment:
It might be better to change `File` to @System.IO.File so that it's resolved as a link.